### PR TITLE
Remember video info for each video

### PR DESCRIPTION
**Pull request by missionfloyd**
This is something I previously implemented in `helloworld.py`. 

The first time a given video is scanned, the results are added to a dict. Then, if the same file is encountered again, the video info is retrieved from the dict rather than running ffprobe again, which can take a few seconds on slower devices.

This'll be particularly useful if a "random frames from folder" mode is added (this is what `helloworld.py` does).

### CODE REVIEW

File: slowmovie.py [hunk 1]
```diff
@@ -181,6 +187,7 @@ def video_info(file):
 width = epd.width
 height = epd.height
 
+videoInfos = {}
```
Comment (qubist) [inline slowmovie.py:190]: What's this line for?

Comment (missionfloyd) [reply slowmovie.py:190]: That's a global variable to keep the video info for each scanned file. It just has to be somewhere before the first `video_info()` call.

**Review by robweber — APPROVED** [review]
I like it. No reason to keep probing a video on each update when we know the valid information.

File: slowmovie.py (hunk 1 above)
Comment (qubist) [reply slowmovie.py:190]: Ah, I see now! Thanks for the explanation 👍

**Review by qubist — APPROVED** [review]

**Pull request merged**